### ISSUE TITLE
API/Resource level throttling does not works for unsecured apis/resources

### ISSUE THREAD

**Issue by Rajith90** [wso2/product-microgateway#1099]
### Description:
If the api is exposed with disabled security and applied resource/api level throttling, throttling will not apply. This scenario is producible for local as well as distributed throttling as well.

### Steps to reproduce:
Add the following extensions to the swagger.

```yaml
x-wso2-throttling-tier: 3PerMin
x-wso2-disable-security: true
```
Then requests will not be throttled when used local or global throttling
### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---

**Comment by Rajith90** [comment]
This fix is merged to 3.1.0 beta

**Comment by praminda** [comment]
What was the PR? Can we close the issue if the fix was merged/
